sympy/algebras/quaternion.py

```diff
@@ -25,6 +25,13 @@
     from sympy.integrals.integrals import SymbolLimits
     SExpr = Expr | complex
 
+def wrap_angle(angle):
+    if isinstance(angle, Number):
+        return (angle + pi) % (2*pi) - pi
+    if isinstance(angle, Expr) and angle.is_number:
+        return (angle + pi) % (2*pi) - pi
```

Comment:
```suggestion
    if isinstance(angle, Expr) and angle.is_number:
        return (angle + pi) % (2*pi) - pi
```

Reply:
S(3).is_Number and S(3).is_number; since the same thing is happening either way, perhaps the single `if` is sufficient?